cloud-pubsub/deployment/pubsub-hpa.yaml

```diff
@@ -17,3 +32,4 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: pubsub
+# [END gke_pubsub_hpa]
```

Comment:
nit, I'd expand any acronyms in region tags to make it easier for someone not familiar with K8s to better understand the region tags in devrel tools

Reply:
`s/hpa/horizontal_pod_autoscaler`